src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheckExamplesTest.java

```diff
@@ -34,36 +33,29 @@ protected String getPackageLocation() {
     @Test
     public void testExample1() throws Exception {
         final String[] expected = {
-
+            "17:23: " + getCheckMessage(MSG_KEY, "a href"),
         };
 
-        verifyWithInlineConfigParser(getPath("Example1.txt"), expected);
+        verifyWithInlineConfigParser(getPath("Example1.java"), expected);
     }
 
     @Test
     public void testExample2() throws Exception {
         final String[] expected = {
-
-        };
-
-        verifyWithInlineConfigParser(getPath("Example2.txt"), expected);
```

Comment:
why just removed ?

Reply:
I removed it, but the git treated it like an edit

Reply:
Please restore it, or explain why you think it should not exists

Reply:
Oh, that's a mistake. I will restore it